src/test/java/de/jetwick/snacktory/ArticleTextExtractorTest.java

```diff
@@ -893,10 +890,10 @@ public void testHuffingtonpostAuthor() throws Exception {
         JResult res = extractor.extractContent(c.streamToString(getClass().getResourceAsStream("huffingtonpost2.html")));
         assertEquals("Millions of Consumers Abandon Hashtag for Backslash", res.getTitle());
         assertTrue(res.getText(), res.getText().startsWith("In a special Silicon Valley \"Tech Report,\" sources confirmed Monday that millions of "));
-        assertEquals("Rebekah Iliff", res.getAuthorName());
+        assertEquals("By Rebekah Iliff", res.getAuthorName());
```

Comment:
Rewrote the CLEAN_AUTHOR_PATTERNS expression to have much cleaner and accurate match.